editor/block-mover/test/mover-label.js

```diff
@@ -23,9 +18,9 @@ describe( 'block mover', () => {
 				true,
 				false,
 				dirUp,
-			) ).to.equal(
-				`Block "${ type }" is at the beginning of the content and can’t be moved up`
-			);
+			) ).toBe(
+                `Block "${ type }" is at the beginning of the content and can’t be moved up`
+            );
```

Comment:
The new indentation here looks wrong.  In fact, I would expect this to cause a lint error and we should investigate why it isn't.